example/mii_core_data_set/generate_cds.py

```diff
@@ -374,6 +374,7 @@ def configure_args_parser():
     arg_parser.add_argument('--generate_ui_profiles', action='store_true')
     arg_parser.add_argument('--generate_mapping', action='store_true')
     arg_parser.add_argument('--generate_old_format', action='store_true')
+    arg_parser.add_argument('--add_delete_statements', action='store_true')
```

Comment:
this will not work as each module has its own generate_modulename.py
=> needs to be eith added as a default true or be part of the 
https://github.com/medizininformatik-initiative/fhir-ontology-generator/blob/develop/example/new_module_template/generate_ontology.py